pkg/logs/tekton_logging_test.go

```diff
@@ -570,19 +636,14 @@ func TestGetRunningBuildLogsWithMultipleStages(t *testing.T) {
 		},
 	}
 
-	logOutput := log.CaptureOutput(func() {
-		err := tl.GetRunningBuildLogs(pa, "abayer/js-test-repo/master/1")
-		assert.NoError(t, err)
-	})
-
-	aORb := regexp.MustCompile("Pod logs...")
-	n := aORb.FindAllStringIndex(logOutput, -1)
+	err := tl.GetRunningBuildLogs(pa, "abayer/js-test-repo/master/1")
```

Comment:
There is a good reason to capture the whole log output, the test output get silent and does not pollute the logs.